src/ui/components/viewControl.vue

```diff
@@ -22,7 +22,12 @@
 <template>
   <span
     :class="[controlClass, { 'c-disclosure-triangle--expanded': value }, { 'is-enabled': enabled }]"
+    tabindex="0"
```

Comment:
Note: this is not 100% correct, but I'm not practiced with tabIndex yet

Reply:
I _think_ it should be fine, if it's got "0" then I think it just goes in order of how the elements show up in the DOM, which for this case should be right.